blockmanager.go

```diff
@@ -1337,12 +1386,17 @@ func newBlockManager(s *server, indexManager blockchain.IndexManager) (*blockMan
 		headerList:      list.New(),
 		quit:            make(chan struct{}),
 	}
-
+	// merge given checkpoints with the default ones unless they are disabled
```

Comment:
Separate this block of code with blank lines for consistency please.

```Go
...
}

// Merge given checkpoints with the default ones unless they are disabled.
var checkpoints []chaincfg.Checkpoint
if !cfg.DisableCheckpoints {
	checkpoints = mergeCheckpoints(s.chainParams.Checkpoints, cfg.checkpoints)
}

// Create a new block chain instance with the appropriate configuration.
...
```

Also, notice I made the comment style consistent.